lib/subfont.js

```diff
@@ -16,17 +17,55 @@ module.exports = async function subfont(
     inlineFonts = false,
     inlineCss = false,
     fontDisplay = 'swap',
-    formats = ['woff2', 'woff'],
+    formats,
+    jsPreload,
     subsetPerPage = false,
     inPlace = false,
     inputFiles = [],
     recursive = false,
     fallbacks = true,
     dynamic = false,
     harfbuzz = false,
+    browsers,
   },
   console
 ) {
+  let selectedBrowsers;
+  if (browsers) {
+    selectedBrowsers = browsersList(browsers);
+  } else {
+    // Will either pick up the browserslist config or use the defaults query
+    selectedBrowsers = browsersList();
+  }
+
+  if (!formats) {
+    formats = ['woff2'];
+    if (
+      _.intersection(
+        browsersList('supports woff, not supports woff2'),
```

Comment:
I haven't seen this interface before. That's pretty awesome!

Reply:
Yeah, it's pretty nifty. A shame that we have to use lodash to combine it with the user's query though.